AlpsSDK/Context Manager/ContextManager.swift

```diff
@@ -48,10 +46,8 @@ public class ContextManager: NSObject, CLLocationManagerDelegate {
     }
 
     public func locationManager(_ manager: CLLocationManager, didRangeBeacons beacons: [CLBeacon], in region: CLBeaconRegion) {
-        delegate?.contextManager(manager: self, didDetectBeacons: beacons)
-        proximityHandler?.didRangeBeacons(manager: self, beacons: beacons, knownBeacons: knownBeacons)
-        if let closestBeacon = beacons.max(by: { $0.accuracy < $1.accuracy }) {
-            delegate?.contextManager(manager: self, didRangeClosestBeacon: closestBeacon)
+        beaconTriples.findAll { result in
+            self.proximityHandler?.didRangeBeacons(manager: self, beacons: beacons, knownBeacons: result.responseObject)
         }
```

Comment:
Be carefule with this. 
DidRangeBeacons is called every 1 seconds. Not sure it's good to have findAll there

Reply:
good catch.